WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductRowCard.swift

```diff
@@ -125,19 +125,33 @@ struct CollapsibleProductRowCard: View {
         }
         .cornerRadius(Layout.frameCornerRadius)
     }
+
+
+}
+
+private extension CollapsibleProductRowCard {
+    func trackAddDiscountTapped() {
+        viewModel.analytics.track(event: .Orders.productDiscountAddButtonTapped())
+    }
+
+    func trackEditDiscountTapped() {
+        viewModel.analytics.track(event: .Orders.productDiscountEditButtonTapped())
+    }
```

Comment:
We could have used only one general method rather than two, as well as call the service locator directly rather than injecting analytics into the view model (since at the moment there's no public testable methods). But this way made it more readable and offloads any non-view-rendering responsibility to the view model.